src/js/components/Chart.vue

```diff
@@ -0,0 +1,341 @@
+      );
+    },
+    sga_delta() {
+      return (
+        this.inputData.total_operating_expenses -
+        this.inputData.total_operating_expenses_y1
+      );
+    },
+    cogs_pct_0() {
+      return (
+        this.inputData.total_cost_of_revenue_y1 /
+        this.inputData.total_revenue_y1
+      );
+    },
+    sga_pct_0() {
+      return (
+        this.inputData.total_operating_expenses_y1 /
+        this.inputData.total_revenue_y1
+      );
+    },
+    cogs_pct_delta() {
+      return this.cogs_delta / this.rev_delta;
+    },
+    sga_pct_delta() {
+      return this.sga_delta / this.rev_delta;
+    },
+    h_cogs_0() {
+      return this.HEIGHT * this.cogs_pct_0;
+    },
+    h_sga_0() {
+      return this.HEIGHT * this.sga_pct_0;
+    },
+    h_cogs_delta() {
+      return this.HEIGHT * this.cogs_pct_delta;
+    },
+    h_sga_delta() {
+      return this.HEIGHT * this.sga_pct_delta;
+    },
+    w_delta() {
+      return this.inputData.total_revenue / this.inputData.total_revenue_y1 - 1;
+    },
+    expenses_0() {
+      return this.h_cogs_0 + this.h_sga_0;
+    },
+    expenses_delta() {
+      return this.h_cogs_delta + this.h_sga_delta;
+    },
+    scale() {
+      return (
+        this.HEIGHT /
+        Math.max(this.expenses_0, this.expenses_delta, this.HEIGHT)
+      );
+    },
+  },
+  methods: {
+    formatUSD(input) {
+      return formatCurrency(input);
+    },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+.chart-container {
+  display: grid;
+  gap: var(--shim);
+  grid-template-columns: min-content 2px auto;
+  margin: var(--double-gutter) var(--gutter);
+}
+
+.revenue-label {
+  align-self: start;
+  color: var(--text);
+  grid-column: 1;
+  margin-top: var(--shim);
+  writing-mode: sideways-lr;
+
+  @supports not (writing-mode: sideways-lr) {
+    text-align: right;
+    transform: rotate(180deg);
+    writing-mode: vertical-rl;
+  }
+}
+
+.revenue-line {
+  grid-column: 2;
+}
+
+line {
+  stroke: var(--text);
+  stroke-width: 3px;
+}
+
+[data-chart] {
+  grid-column: 3;
+}
+
+rect {
+  stroke: var(--rect-stroke);
+  stroke-width: 2px;
+}
+
+.income {
+  fill: var(--income);
+}
+
+.sga {
+  fill: var(--sga);
+}
+
+.cogs {
+  fill: var(--cogs);
+}
+
+.corner {
+  fill: #ffffff;
+}
+
+[data-table] {
+  border-collapse: collapse;
+  border-radius: 4px;
+  box-shadow: 1px 3px 6px #00000029;
+  color: var(--text);
+  font-family: inherit;
+  grid-column: 3;
+  max-width: 30rem;
+  width: 100%;
+
+  th {
+    font-size: 0.75rem;
+    font-weight: bold;
+  }
+
+  th,
+  td {
+    border-bottom: thin solid var(--border-color);
+    padding: var(--gutter);
+  }
+}
+
+tbody {
+  font-size: 0.875rem;
+}
+
+[data-cell~='type'] {
+  text-align: left;
+}
+
+[data-cell~='type']::before {
+  background-color: var(--type-color);
+  border-radius: 100%;
+  content: '';
+  display: inline-block;
+  height: 14px;
+  margin-right: var(--shim);
+  width: 14px;
+}
+
+[data-cell~='cogs']::before {
```

Comment:
```suggestion
[data-cell~='cogs'] {
```

Very minor, but we don't need to target the pseudo, since these will inherit… (same on the other two)

Reply:
removed these